server/command/rotation_show.go

```diff
@@ -3,25 +3,27 @@
 
 package command
 
-func (c *Command) showRotation(parameters []string) (string, error) {
-	var rotationID, rotationName string
-	fs := newRotationFlagSet(&rotationID, &rotationName)
+import "github.com/mattermost/mattermost-plugin-solar-lottery/server/utils/md"
+
+func (c *Command) rotationShow(parameters []string) (md.MD, error) {
+	fs := c.assureFS()
```

Comment:
That's because all other c.fXXX() functions initialize c.fs lazily.

2/5 maybe replace pflag.Parse with a c.Parse and lazily init `fs` there. Will do.